third_party/move/move-compiler/src/expansion/translate.rs

```diff
@@ -3054,7 +3100,7 @@ fn assign(context: &mut Context, sp!(loc, e_): P::Exp) -> Option<E::LValue> {
             )?;
             let tys_opt = optional_types(context, ptys_opt);
             let efields = assign_unpack_fields(context, loc, pfields)?;
-            EL::Unpack(en, tys_opt, efields)
+            EL::Unpack(en, tys_opt, efields, None)
```

Comment:
Brief comment please (for the use of `None`).

Reply:
done